Models/comments-model.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection');
+
+exports.selectCommentsByArticleId = id => {
+  return db
+    .query(
+      `SELECT * FROM comments
+       WHERE article_id = $1
+       ORDER BY created_at ASC`,
+      [id]
+    )
+    .then(({ rows }) => {
+      if (!rows.length) {
```

Comment:
This will need modifying to take care of the functionality that needs testing for as mentioned above.